src/DailyLeetcodeReminder.Infrastructure/Jobs/DailyReportJob.cs

```diff
@@ -121,19 +121,24 @@ public async Task Execute(IJobExecutionContext context)
 
     private string GetDailyReportDetails(List<Challenger> activeChallengers)
     {
+        int maximalUsername = 0;
+        foreach (var username in activeChallengers)
+        {
+            maximalUsername = Math.Max(username.LeetcodeUserName.Length, maximalUsername);
```

Comment:
`int mostLongestUsernameLength = activeChallengers.Max(ch => ch.LeetcodeUserName.Length);` check it please